### PR TITLE
Fix disable_transitions.js injection so transitions actually get disabled

### PR DESCRIPTION
**Pull request by gigxz**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

The previous code was not actually disabling transitions (confirmed in debug mode). This change appears to actually disable them

## Type of change
[//]: # 'remove options that are not relevant'
- [x] Bug fix
## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: bin/run_hmis_system_tests.sh [hunk 1]
```diff
@@ -108,6 +108,11 @@ cd "$ORIGINAL_CWD"
 unset HMIS_OKTA_CLIENT_ID
 unset OKTA_DOMAIN
 RUN_SYSTEM_TESTS=true RAILS_ENV=test CAPYBARA_APP_HOST="http://$HOSTNAME:5173" rspec drivers/hmis/spec/system/hmis/*
+RUN_SYSTEM_TESTS=true RAILS_ENV=test CAPYBARA_APP_HOST="http://$HOSTNAME:5173" rspec drivers/hmis/spec/system/hmis/*
+RUN_SYSTEM_TESTS=true RAILS_ENV=test CAPYBARA_APP_HOST="http://$HOSTNAME:5173" rspec drivers/hmis/spec/system/hmis/*
+RUN_SYSTEM_TESTS=true RAILS_ENV=test CAPYBARA_APP_HOST="http://$HOSTNAME:5173" rspec drivers/hmis/spec/system/hmis/*
+RUN_SYSTEM_TESTS=true RAILS_ENV=test CAPYBARA_APP_HOST="http://$HOSTNAME:5173" rspec drivers/hmis/spec/system/hmis/*
+RUN_SYSTEM_TESTS=true RAILS_ENV=test CAPYBARA_APP_HOST="http://$HOSTNAME:5173" rspec drivers/hmis/spec/system/hmis/*
```
Comment (gigxz) [inline bin/run_hmis_system_tests.sh]: leaving in draft mode while these run, then will remove. Had several successful runs but just doing a few more for peace of mind

File: drivers/hmis/spec/system/hmis/assessment_definitions_spec.rb [hunk 2]
```diff
@@ -181,17 +179,17 @@ def select_member(client)
...
 
     def unlock_household_assessment
-      find('button', text: 'Unlock Assessment').trigger('click')
+      click_button 'Unlock Assessment'
       assert_text 'Save & Submit' # Unlock succeeded
     end
 
     def submit_household_assessment
-      find('button', text: 'Save & Submit').trigger('click') # note: using click_button fails on CI (#6619)
+      click_button 'Save & Submit'
```
Comment (gigxz) [inline drivers/hmis/spec/system/hmis/assessment_definitions_spec.rb:187]: Can easily reproduce the fix here. The `SaveSlide` component at the bottom of the assessment was sliding in, which was causing click_button not to work. Now that the transitions are disabled, it no longer slides in, so this works

**Comment by gigxz** [comment]
Note that `ReferenceError: _cuprite is not defined` still appears, this fixes a separate issue.
https://github.com/greenriver/hmis-warehouse/actions/runs/11002006286/job/30547991954

**Review by ttoomey — APPROVED** [review]
Great, thanks for tracking that down! The `extensions` option seems much cleaner too

**Review by martha — APPROVED** [review]
Great 🙏

**Pull request merged**